### PR TITLE
fix wrong parse operation when service url is short

### PR DESCRIPTION
**Pull request by XiaoJiangWang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug

> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

fix wrong parse operation when service url is short in edgemesh load balancer chain

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edgemesh/pkg/plugin/registry/registry.go [hunk 1]
```diff
@@ -141,7 +140,9 @@ func parseServiceURL(serviceURL string) (name, namespace string, port int, err e
 	} else {
 		klog.Errorf("[EdgeMesh] service url %s invalid", serviceURL)
 		err = fmt.Errorf("service url %s invalid", serviceURL)
+		return
```
Comment (fisherxu) [inline edgemesh/pkg/plugin/registry/registry.go:143]:
```suggestion
		return fmt.Errorf("service url %s invalid", serviceURL)
```
And the error in L137 also need return :)

Comment (XiaoJiangWang) [reply edgemesh/pkg/plugin/registry/registry.go:143]: It is not necessary. In L129, the function definition showed the return values.

Comment (fisherxu) [reply edgemesh/pkg/plugin/registry/registry.go:143]: We can delete the unneeded return values.

Comment (XiaoJiangWang) [reply edgemesh/pkg/plugin/registry/registry.go]: fixed

**Comment by fisherxu** [comment]
/lgtm
It seems that the email address you used in your commit hasn't been added to your github personal profile. So the contributions will not be counted in github. :)
image

**Comment by XiaoJiangWang** [comment]
> /lgtm
> It seems that the email address you used in your commit hasn't been added to your github personal profile. So the contributions will not be counted in github. :)
> image

fixed. thanks.

**Pull request merged**